tests/regress/issue18_test.toit

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
The copyright headers in (some of) these files look a bit weird and poorly formatted.

We should change these to have the 0BSD license information and clean them up.

Reply:
Agreed.
Not in this PR, though. (Too many dependent PRs right now).